cookietemple/create/github_support.py

```diff
@@ -207,19 +207,32 @@ def prompt_github_repo(dot_cookietemple: Optional[dict]) -> Tuple[bool, bool, bo
 
     # No dot_cookietemple_dict was passed -> prompt whether to create a Github repository and the required settings
     create_git_repo, private, is_github_org, github_org = False, False, False, ''
-    if cookietemple_questionary_or_dot_cookietemple(function='confirm',
-                                                    question='Do you want to create a Github repository and push your template to it?',
-                                                    default='Yes'):
-        create_git_repo = True
-        is_github_org = cookietemple_questionary_or_dot_cookietemple(function='confirm',  # type: ignore
-                                                                     question='Do you want to create an organization repository?',
-                                                                     default='No')
-        github_org = cookietemple_questionary_or_dot_cookietemple(function='text',  # type: ignore
-                                                                  question='Please enter the name of the Github organization',
-                                                                  default='SpringfieldNuclearPowerPlant') if is_github_org else ''
-        private = cookietemple_questionary_or_dot_cookietemple(function='confirm',  # type: ignore
-                                                               question='Do you want your repository to be private?',
-                                                               default='No')
+    while True:
```

Comment:
This looks really ugly.

Why are all of these workarounds necessary?

Just add a second confirm prompt if the second one was declined.

Reply:
No, I dont want the application to end, instead give the opportunity for remake the decision